server.js

```diff
@@ -92,8 +93,9 @@ var staticServer = new nodeStaticAlias.Server(STATIC_DIR,{
 	],
 });
 
-httpServer.listen(8034);
+httpServer.listen(PORT);
 
+console.log(`The app is running on http://localhost:${PORT}`);
```

Comment:
Wouldn't be better to add this `console.log` in the `listen` callback? 
Example: 

```js
httpServer.listen(PORT, () => {
    console.log(`The app is running on http://localhost:${PORT}`);
});
```

This way, the message will be only logged if the server started with success.

Reply:
Great suggestion.
I'll update it.